test/index.js

```diff
@@ -9,7 +9,9 @@ test('Tests run', (assert) => {
   assert.end();
 });
 
-test('Greet World', (assert) => {
+
+test('Greet World', (assert) => new Promise((resolve) => {
```

Comment:
Apparently `no-new` eslint rule restricts `new` for side-effects. So you can't just put `new Something()` without assigning it or returning. It is allowed to `return new Promise()` or `const promise = new Promise()`.